testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/jsonb/basic/JsonBindingTest.java

```diff
@@ -137,6 +137,31 @@ public void jsonbOnClientTest() throws Exception {
              response.getTransientVar(), is(Cat.DEFAULT_TRANSIENT_VAR_VALUE));
    }
 
+   /**
+    * @tpTestDetails JSON-B is used on client, JSON-B is not used on server, server uses test's custom json provider
+    *                client should not ignore @JsonbTransient annotation and should not send a value in this variable
+    *                server verify that client doesn't sent a value in a variable with @JsonbTransient annotation
+    *                server returns json data with a value in a variable with @JsonbTransient annotation
+    *                client should not ignore @JsonbTransient annotation and should not receive a value in this variable
+    *
+    *                This test covers RESTEASY-2171.
+    *
+    * @tpPassCrit The resource returns object with correct values
+    * @tpSince RESTEasy 3.5
+    */
+   @Test
+   public void jsonbOnClientTestWithoutEncoding() throws Exception {
```

Comment:
This test doesn't check non-UTF8 encoding. Test needs to be fixed

Reply:
It shouldn't be tested as the fix is for the specific case of the encoding not being provided.

Reply:
Well, I add this my comment 3 days ago and I believe that this is fixed now, new container uses us-ascii, see https://github.com/resteasy/Resteasy/pull/1910/files#diff-49db9a69f91d736f22ab33486a4c65bbR36

Btw, If the fix is for some specific case, tests should checks just this specific case ...